src/Package.js

```diff
@@ -96,6 +96,9 @@ export default class Package {
     // check if semantic versions are compatible
     if (semver.satisfies(actualVersion, expectedVersion)) {
       return true;
+    // or the dependency is specified as a relative folder
+    } else if (/^file:\.\.\//.test(dependency.version)) {
+      return true;
```

Comment:
bootstrapping would fail with an npm error as it should. I don't think it should be the job of this tool to detect whether someone has misconfigured their package.json